README.md

```diff
@@ -1,192 +1,165 @@
-Dversion=<version> -Dpackaging=jar`
+Run `./install-deps.sh` to install dependencies not in a maven repo. This script copies dependencies from lib/ into
+your local maven repo.
 
 After running the two commands, right click on the Measure Authoring Tool project -> Maven -> Update Project
 (ensure force update is checked) -> Ok.
 
 ### Create MAT Database
-1.	Install MySQL (MAT currently uses MySQL Community Version 6.x) available from [MySQL](https://dev.mysql.com/downloads/installer/)
-2.	Run the MySQL community server installer for your operating system and the MySQL workbench (which comes with the download).
-3.	Create a new MySQL Connection to the database.
-Note: Make sure to keep track of the username, password, url, and port you used as you will need this later.
-4.	From the MAT Code base, find the `mat_schema.sql` file and then execute the script in the database that was just created.
-
-### Configure Files for Development
-Changes will need to be made to some of the files in the code base to allow the MAT to connect to the MySQL Database.
-
-#### add `mat/war/WEB-INF/jetty-env.xml`
-```xml
-<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure.dtd">
-<Configure id="Server" class="org.eclipse.jetty.webapp.WebAppContext">
-<New id="MatJndi" class="org.eclipse.jetty.plus.jndi.Resource">
-  <Arg></Arg>
-  <Arg>java:/comp/env/jdbc/mat_app_tomcat</Arg>
-  <Arg>
-    <New class="com.mysql.jdbc.jdbc2.optional.MysqlConnectionPoolDataSource">
-        <Set name="User">username</Set>
-        <Set name="Password">password </Set>
-        <Set name="DatabaseName">schemaname</Set>
-        <Set name="ServerName">serveraddress</Set>
-        <Set name="PortNumber">port</Set>
-      </New>
-    </Arg>
-</New>
-</Configure>
-```
-If you get exception as below `'com.google.gwt.dev.shell.jetty.JettyLauncher.WebAppContextWithReload'` is not of type `'org.eclipse.jetty.webapp.WebAppContext'` then you will need to find a way to load the jetty class instead of `gwt-dev.jar`. One way to make it work is to change the `gwt-dev` dependency in the `pom.xml` to be `provided` rather than `compile`. If you do this, then when running `mvn clean install` you will have to change it back.
-```xml
-<dependency>
-    <groupId>com.google.gwt</groupId>
-    <artifactId>gwt-dev</artifactId>
-    <version>${gwtVersion}</version>
-    <scope>provided</scope>
-    <exclusions>
-        <exclusion>
-            <artifactId>commons-collections</artifactId>
-            <groupId>commons-collections</groupId>
-        </exclusion>
-        <exclusion>
-            <groupId>org.eclipse.jetty</groupId>
-            <artifactId>apache-jsp</artifactId>
-        </exclusion>
-    </exclusions>
-</dependency>
+- Install MySQL (MAT currently has been tested with MySQL Community Version 8.0+) available from [MySQL] (https://dev.mysql.com/downloads/mysql/)
```

Comment:
I think it was "MySQL (MAT currently has been tested with MySQL Community Version **5.7**)" 
I'm not aware of plans to upgrade MySQL to 8.0+
Do you have a confirmation from @AndrewBird81  or @nicole-hunter ?

Reply:
I am not checking in the driver changes for the pom.xml actually included in the directions not to check that in. These are just instructions on how to run it locally. I gave up trying to get 5.7 working in osx. The latest version works if you change the driver but obviously I haven't done end to end testing on it.

Reply:
Hm... interesting.. I was able to install MySQL 5.7.28 with brew on my Mac. I don't recall any issues...